src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsExtensions.cs

```diff
@@ -25,32 +25,40 @@ namespace Dynatrace.OpenTelemetry.Exporter.Metrics
 	internal static class DynatraceMetricsExtensions
 	{
 		public static DynatraceMetric ToLongCounterDelta(this Metric metric, MetricPoint metricPoint, ILogger logger)
-			=> DynatraceMetricsFactory.CreateLongCounterDelta(
+		{
+			return DynatraceMetricsFactory.CreateLongCounterDelta(
```

Comment:
```suggestion
			if (metric.Temporality != AggregationTemporality.Delta)
			{
				logger.LogWarning($"Non-Delta Sum encountered, dropping... ({metric.Name})");
				return null;
			}
			return DynatraceMetricsFactory.CreateLongCounterDelta(
```

Reply:
With the latest changes (making the exporter internal), I think there's no way to get cumulative values here. We might also want to mark these extension methods internal just to be consistent and completely "shielded".

Reply:
I agree with Joao on this. I think there is no way to get a different temporality as long as the exporter is `internal`. :thinking: 

However, we agreed offline to add something like this once the need arises to make the exporter `public`. :slightly_smiling_face: